### DISCUSSION TITLE
dependency or conflict

### DISCUSSION THREAD

**Discussion by WWWNoobhere** [microsoft/winget-cli#2531] [Q&A]
hi sorry for bothering you i get this error while tried to install the 1.4.2161

PS C:\Windows\system32> add-appxpackage -path "C:\Users\mahdi\Desktop\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe (1).msixbundle"
add-appxpackage : Deployment failed with HRESULT: 0x80073CF3, Package failed updates, dependency or conflict
validation.
Windows cannot install package Microsoft.DesktopAppInstaller_1.19.2161.0_x64__8wekyb3d8bbwe because this package
depends on a framework that could not be found. Provide the framework "Microsoft.UI.Xaml.2.7" published by
"CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor
architecture and minimum version 7.2109.13004.0, along with this package to install. The frameworks with name
"Microsoft.UI.Xaml.2.7" currently installed are: {}
NOTE: For additional information, look for [ActivityId] 4721a19e-cb76-0002-c802-224776cbd801 in the Event Log or use
the command line Get-AppPackageLog -ActivityID 4721a19e-cb76-0002-c802-224776cbd801
At line:1 char:1
+ add-appxpackage -path "C:\Users\mahdi\Desktop\Microsoft.DesktopAppIns ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (C:\Users\mahdi\... (1).msixbundle:String) [Add-AppxPackage], IOException
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand
    
    can anyone please help me solve this problem...

**Comment by NLZ** [comment]
Install Microsoft.UI.Xaml.2.7:

```pwsh
Invoke-WebRequest -Uri https://www.nuget.org/api/v2/package/Microsoft.UI.Xaml/2.7.3 -OutFile .\microsoft.ui.xaml.2.7.3.zip
Expand-Archive .\microsoft.ui.xaml.2.7.3.zip
Add-AppxPackage .\microsoft.ui.xaml.2.7.3\tools\AppX\x64\Release\Microsoft.UI.Xaml.2.7.appx
```
From https://github.com/microsoft/winget-cli/issues/1861#issuecomment-1352784247
